Source/WebCore/html/PluginDocument.cpp

```diff
@@ -76,7 +76,11 @@ Ref<HTMLStyleElement> PluginDocumentParser::createStyleElement(Document& documen
 {
     auto styleElement = HTMLStyleElement::create(document);
 
-    constexpr auto styleSheetContents = "html, body, embed { width: 100%; height: 100%; }\nbody { margin: 0; overflow: hidden; }\n"_s;
+    constexpr auto styleSheetContents = R"CONTENTS(
+html, body, embed { width: 100%; height: 100%; }
+body { margin: 0; overflow: hidden; }
+html.plugin-fits-content body { overflow: revert; }
+)CONTENTS"_s;
```

Comment:
Can we indent this?

Reply:
Indented in https://github.com/WebKit/WebKit/commit/e6abaffdcea753bb1711d92bc067f42526af4c96